src/commonmark/en/content/developer/web-api/metadata.md

```diff
@@ -2315,6 +2316,43 @@ the following resource.
 </tbody>
 </table>
 
+### Get organisation units by programs
+
+<!--DHIS2-SECTION-ID:webapi_organisation_units_by_programs-->
+
+There's a special ad-hoc endpoint built to retrieve associations between Programs and Organisation Units. This should be
+the preferred way to retrieve programs that are either:
+
+- unassigned to any organisation units 
+- assigned explicitly to org units within the scope of logged in user capture scope.
+
+
+    /api/33/programs/orgUnits?programs=<programUid,anotherProgramUid,...>
+
+responses will have the following format:
+
+    {
+        "<programUid>": [
+            "<orgUnitUid_1>",
+            "<orgUnitUid_2>",
+            ...,
+            "<orgUnitUid_n>"
+        ],
+        "<anotherProgramUid>": [
+            "<orgUnitUid_1>",
+            "<orgUnitUid_2>",
+            ...,
+            "<orgUnitUid_n>"
+        ],
+        "...": [
+            ...,
+            ...
+        ],
+        "<yetAnotherProgramUid>": []
+    }
+
+programs with no associated uids (hence accessible by all orgUnits) are returned with an empty array [] of orgUnits.
```

Comment:
Question not directly relevant to the docs, but to the code - is this the same functionality that we see in the old programs endpoint?

Reply:
when the old endpoint was invoked to retrieve program/org units associations (with restrictToCaptureScope=true), the response was the same, in the meaning that for each program, the two endpoints returns the same orgUnits associations. The shape of the responses is different though.